forte/data/data_store.py

```diff
@@ -200,10 +196,28 @@ def get_attr(self, tid: int, attr_id: int):
         Returns:
             The value of `attr_id` for the entry with `tid`.
         """
-        # We retrieve the entry data from entry_dict using tid. We get its
-        # entry type. We then locate the attribute using attr_id, and get the attribute.
-
-        raise NotImplementedError
+        # We retrieve the entry data from entry_dict using tid.
+        # We then locate the attribute using attr_id, and get the attribute.
+        entry = self.entry_dict[tid]
+        return entry[attr_id]
+
+    def set_attribute(self, tid: int, attr_name: str, attr_value):
+        if tid not in self.entry_dict.keys():
+            raise ValueError(f"Entry with tid {tid} not found.")
+        entry_type = self.entry_dict[tid][3]
```

Comment:
It is better not to hardcode the position of the items in code. Create a few consts in this file and use them here.